vtex-sites/gatsby.store
# fix: collection pages facets

## What's the purpose of this pull request?

More info at: https://github.com/vtex/faststore/pull/1477

## How does it work?

- Fix the search facets to use the productClusterIds when the pageType is Collection

## How to test it?

- ❌ Navigate to page: [https://gatsby.vtex.app/just-arrived](https://gatsby.vtex.app/just-arrived) and you will see that it returns no search result
- ✅ Navigate to page: [https://sfj-e15d07e--gatsby.preview.vtex.app/just-arrived](https://sfj-e15d07e--gatsby.preview.vtex.app/just-arrived) and you will see that it works

https://github.com/vtex-sites/arcaplanet.store/pull/30 <<< Store that needed this PR, can be tested by navigating to `/novita` or clicking on the following link:
- https://sfj-d7e803f--arcaplanet.preview.vtex.app/novita

## References

<em>Spread the knowledge: is this any content you used to create this PR that is worth sharing?</em>

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>
